tokio-trace-subscriber/src/registry.rs

```diff
@@ -171,17 +171,15 @@ impl RegisterSpan for IncreasingCounter {
 
     fn new_span(&self, _new_span: &Attributes) -> Id {
         let id = self.next_id.fetch_add(1, Ordering::SeqCst);
-        let id = Id::from_u64(id as u64);
+        Id::from_u64(id as u64)
         // if let Ok(spans) = self.spans.lock() {
```

Comment:
the commented out code should probably also be removed; this is why we had the (now unnecessary) let binding...